helios/pipeViewer/pipe_view/model/schedule_element.py

```diff
@@ -387,6 +389,8 @@ def GetQueryFrame(self, period: int) -> Tuple[int, int]:
     def DetectCollision(self,
                         pt: Union[Tuple[int, int], wx.Point],
                         pair: Element_Value) -> FakeElementValue:
+        from .element_value import FakeElementValue
+
```

Comment:
`FakeElementValue` is only actually used in this function. By importing it here, we avoid a circular import dependency with `element_value.py`.